### PR TITLE
Remove CropWhitespace function

### LINKED ISSUE #4120: Logo image for artists is cropped when uploaded. Not CSS issue
**Describe the bug**
When adding a logo image to an artist it crops the image slightly.

**System (please complete the following information):**
 - OS: Debian
 - Virtualization: None
 - Clients: Web
 - Browser: Safari 16610.2.2.4
 - Jellyfin Version:10.6.2
 - Playback: N/A
 - Installed Plugins: None
 - Reverse Proxy: N/A
 - Base URL: n/a
 - Networking: n/a
 - Storage: local

**To Reproduce**

1. Go to and artist and select edit images. upload 

2. The logo edges have been cropped

**Expected behavior**
Not have logo edges cropped. It's not the css either. the actual image is returned cropped. There are a few other examples I can show if needed.

### PR DESCRIPTION
**Pull request by Bond-009**
Fixes #4120

### CODE REVIEW

File: Jellyfin.Api/Controllers/ImageController.cs [hunk 1]
```diff
@@ -1681,7 +1681,7 @@ private static async Task<MemoryStream> GetMemoryStream(Stream inputStream)
             int? width,
             int? height,
             int? quality,
-            bool? cropWhitespace,
+            bool? cropWhitespace, // TODO: Remove
```
Comment (nielsvanvelzen) [inline Jellyfin.Api/Controllers/ImageController.cs:1684]: Can you mark it deprecated in the other functions so it propagates to the OpenAPI spec?

Comment (crobibero) [reply Jellyfin.Api/Controllers/ImageController.cs:1684]: I don't see a way to mark it as deprecated. I think the parameter should just be removed if it's just ignored now.

**Review by barronpm — APPROVED** [review]

File: Jellyfin.Api/Controllers/ImageController.cs (hunk 1 above)
Comment (Chobocharles) [reply Jellyfin.Api/Controllers/ImageController.cs:1684]: @crobibero I know I'm super late to the party, but you can utilize the ObsoleteAttribute class for deprecated/obsolete items:
`[Obsolete("Enter whatever message you'd like.")]`

Comment (crobibero) [reply Jellyfin.Api/Controllers/ImageController.cs:1684]: Last time I tested, we weren’t able to put the `Obsolete` attribute on a parameter. You can do it on the entire controller or specific endpoints, which is what we already do.

**Review by cvium — APPROVED** [review]

**Pull request merged**